examples/website/src/shared/route/articleTemplateContainer.tsx

```diff
@@ -134,16 +134,18 @@ export function ArticleTemplateContainer({id, slug}: ArticleTemplateContainerPro
   } = articleData
 
   const path = ArticleRoute.reverse({id, slug})
-  const articleCanonicalURL = canonicalUrl || canonicalHost + path
+  const canonicalURL = canonicalHost + path
+  const canonicalPeerURL = canonicalUrl || canonicalHost + path
```

Comment:
this is very bad variable naming. You have a `canonicalURL` and a `canonicalUrl`. Please change this.

Reply:
@roosnic1 good point, I changed it to be canonicalOwnURL. What do you think?